examples/chart/teleport-kube-agent/templates/deployment.yaml

```diff
@@ -214,6 +224,14 @@ spec:
         secret:
           secretName: {{ .Values.tls.existingCASecretName }}
       {{- end }}
+      {{- if contains "jamf" (.Values.roles | toString) }}
+      - name: "jamf-api-credentials"
+        secret:
+          secretName: {{ .Values.jamfCredentialsSecret.name }}
+          items:
+          - key: jamfSecret
+            path: secret
```

Comment:
The term secret is already confusing because we have a jamf client secret sotred in a kubernmetes sxecret, but can also have a jamf password stored in the kube secret. Naming the key jamfSecret makes it look like it always contains the jamfClientSecret.

You came up with the term "credential" which covers both, I think it would be clearer to use it both as a kube scret key, and as the file path.

e.g.
```
          items:
          - key: jamfCredential
            path: credential
```

Or even use the same key and path so you don't have to specify the items in the volume.

Reply:
Thanks for the suggestion! I updated naming to `credential` and simplified volume spec.